### ISSUE TITLE
build.gradle from npm has wrong dependencies

### ISSUE THREAD

**Issue by pasviegas** [xinthink/react-native-material-kit#35]
If I do 'npm install react-native-material-kit', I get the following on build.gradle:

```groovy
dependencies {
    compile project(':ReactAndroid')
//    provided 'com.facebook.react:react-native:0.12.+'
    //provided ('com.facebook.react:react-native:0.11.+') {
    //    exclude module: 'support-v4'
    //    exclude module: 'appcompat-v7'
    //}
    //compile 'com.android.support:design:23.0.1'
}
```
But I see that in the repo it is correct:
```groovy
dependencies {
    provided 'com.facebook.react:react-native:0.12.+'
}
```
Thanks

**Comment by xinthink** [comment]
Sorry about the mistake, npm publish my local copy of the file. Will fix it ASAP

**Comment by xinthink** [comment]
just published a  [v0.2.1](https://github.com/xinthink/react-native-material-kit/releases/tag/v0.2.1) to fix it

**Comment by pasviegas** [comment]
;)
